### ISSUE TITLE
Allow using host DNS with pod annotation

### ISSUE THREAD

**Issue by sandangel** [loft-sh/vcluster#955]
### Is your feature request related to a problem?

Right now we have to find the static host nameserver IP and add that to dnsConfig.nameservers of pod so it can resolve services on host cluster.

### Which solution do you suggest?

Use a specific vcluster annotation so the pod would use host nameserver IP automatically

### Additional context

We discussed in this slack thread: https://loft-sh.slack.com/archives/C01N273CF4P/p1677885954175339

**Comment by FabianKramm** [comment]
@sandangel yes would could add a syncer option to configure extra nameservers.

**Comment by sandangel** [comment]
but then we still have to look for the static IP of host right? the main purpose is that user do not have to look for the cluster static nameserver IP, and that IP can be different depends on the cluster (in case multi cluster connected by service mesh).

**Comment by FabianKramm** [comment]
@sandangel mhh, I see, why do you need this in general? Whats the use case behind this?

**Comment by sandangel** [comment]
it's described in the slack thread.

**Comment by sandangel** [comment]
this is super cool. thanks a lot
